laws/src/main/scala/algebra/laws/LatticeLaws.scala

```diff
@@ -23,6 +23,7 @@ trait LatticeLaws[A] extends Laws {
   def joinSemilattice(implicit A: JoinSemilattice[A]) = new LatticeProperties(
     name = "joinSemilattice",
     parents = Nil,
+    Rules.serializable(A),
```

Comment:
`JoinLattice` (and `MeetLattice`) both don't extend `Serializable`, while the others do. Should those also extend `Serializable`?